.github/lint-disallowed-functions-in-library.sh

```diff
@@ -24,6 +24,7 @@ DISALLOWED_FUNCTIONS=('os.Exit(' 'panic(' 'Fatal(' 'Fatalf(' 'Fatalln(' 'fmt.Pri
 files=$(
   find "$SCRIPT_PATH/.." -name "*.go" \
     | grep -v -e '^.*_test.go$' \
+    | grep -v -e 'waitgroup.go$' \
```

Comment:
Change to this file will be automatically overwritten from pion/.goassets. (See header for details)
I think it's better to add support for `// nolint` comment to pion/.goassets.

Reply:
Thx, fixed 👍

Reply:
ah I forgot `nolint` comment is already supported